content/tutorials/ios-push-server-registration.textile

```diff
@@ -1,12 +1,13 @@
 ---
 title: iOS Push Notifications Tutorial - Device registration via server
+site_title: iOS Push Notification Registration via Server
```

Comment:
Why not Tutorial?

Reply:
On the site it automatically inserts `Tutorial / TITLE`, so felt it was overkill to include `Tutorial` twice.

Reply:
> so felt it was overkill to include Tutorial twice.

Ok, so how will an author of a page in the docs repo know what to expect on the website?  That is my concern with this approach.  I think we need to a) have a similar approach to titling here and on the website, b) have a simple guide in our docs on how to name pages.